src/openforms/conf/base.py

```diff
@@ -995,25 +1005,31 @@
 # CSP_SANDBOX # too much
 
 CSP_UPGRADE_INSECURE_REQUESTS = False  # TODO enable on production?
-# CSP_INCLUDE_NONCE_IN = ["script-src"]  # if we inline we should at least have this
+
+CSP_EXCLUDE_URL_PREFIXES = (
+    # ReDoc/Swagger pull in external sources, so don't enforce CSP on API endpoints/documentation.
+    "/api/",
+    # FIXME: Admin pulls in bootstrap from CDN & has inline styles/scripts probably
```

Comment:
Will this cause a gazillion Sentry issues?

Reply:
no, because CSP is essentially disabled for the admin. No headers are emitted at all for the admin, so there's nothing to report.